dz_s6/from_seminar/__init__.py

```diff
@@ -0,0 +1,8 @@
+__all__ = ['date_exists',
+           'riddle',
+           'riddles_in_dict',
+           'riddles_in_dict_with_results',
+           'secret_number',
+           'secret_number_terminal',
+           'chess',
+           'chess_placement_var']
```

Comment:
В книге "Чистый Python" Дэна Бейдера рекомендуется записывать коллекции в столбик, ка это сделали вы. Кроме того, рекомендуется оставлять запятую после последнего элемента. Это упростить работа с гитом при изменении этой коллекции
```suggestion
__all__ = [
    'date_exists',
    'riddle',
    'riddles_in_dict',
    'riddles_in_dict_with_results',
    'secret_number',
    'secret_number_terminal',
    'chess',
    'chess_placement_var',
]
```